grpc/v1/BUILD.bazel

```diff
@@ -4,11 +4,14 @@ load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 
 proto_library(
     name = "v1_proto",
-    srcs = ["webserver.proto"],
+    srcs = [
+        "query.proto",
+        "webserver.proto",
+    ],
+    strip_import_prefix = "/grpc/v1",
```

Comment:
This was also a new one for me. Apparently this is needed to make proto imports work for both `buf` and `bazel`.